### ISSUE TITLE
Could not pass AsyncPlayerChatEvent

### ISSUE THREAD

**Issue by Benlewis9000** [gronnmann/CoinFlipper#9]
Occurs when any player sends a message to chat. Haven't noticed anything other than this error.

```
[23:03:26 ERROR]: Could not pass event AsyncPlayerChatEvent to CoinFlipper v2.42
org.bukkit.event.EventException
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:499) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.chat(PlayerConnection.java:1111) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1026) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat$1.run(PacketPlayInChat.java:39) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_212]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_212]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_212]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_212]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
Caused by: java.lang.Error: Unresolved compilation problems:
        The constructor ComponentBuilder(TextComponent) is undefined
        The method append(String) in the type ComponentBuilder is not applicable for the arguments (TextComponent)

        at io.github.gronnmann.coinflipper.gui.configurationeditor.messages.MessageEditor.handleEditing(MessageEditor.java:215) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_212]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_212]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_212]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_212]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:300) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        ... 12 more
[23:03:26 INFO]: ***aaa Owner » Anarchist: t

```

**Comment by gronnmann** [comment]
This seems like a widespread issue on some of the new spigot versions, working hard on a fix.

**Comment by Benlewis9000** [comment]
Sorry, forgot to mention this was on a paperspigot 1.8.8 server, running CF 2.42.

**Comment by gronnmann** [comment]
Hmm kinda surprising, maybe PaperSpigot has the new ChatComponent methods. Just made a fix that works for 1.9+, fine tuning it so all versions will be fixed.

**Comment by gronnmann** [comment]
Problem should be fixed in version 2.4_3. Please report back if you still have the issue, if not then I will close the ticket in a week.

**Comment by Benlewis9000** [comment]
Seems to be fixed, thank you!
